app/routines/export_users_last_login_date.rb

```diff
@@ -0,0 +1,53 @@
+class ExportUsersLastLoginDate
+  lev_routine
+
+  protected
+
+  def exec(delete_after: true)
+    outputs[:filename] = filename
+    outputs[:info] = info
+    generate_csv
+    remove_exported_files if delete_after
+  end
+
+  def info
+    output_users = []
+    User.preload(:contact_infos).find_each do |user|
+      output_users << Hashie::Mash.new({
+        emails: user.contact_infos.map(&:value).join(", "),
```

Comment:
IIRC this will include unverified emails, which will not be what we want.

Reply:
Fixed :)